webpack.config.js

```diff
@@ -9,11 +9,14 @@ module.exports = merge(require('./webpack.base'), {
     output: {
         path: path.resolve(__dirname, 'dist'),
         filename: 'index.js',
-        library: 'vue-tabs',
+        library: 'vue-table',
         libraryTarget: 'umd',
     },
 
-    externals: {
-        vue: 'vue',
-    },
+    externals: [
+        'vue',
+        'moment',
+        'lodash',
+        'striptags'
```

Comment:
**striptags** is being used by component here
https://github.com/spatie/vue-table-component/blob/master/src/classes/Row.js#L2
not only by **jest**